<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisaoWidget</class>
 <widget class="QWidget" name="VisaoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="visao_grid">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Referências</string>
      </attribute>
      <layout class="QGridLayout" name="referencias_grid">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="tipo_ref_lbl">
         <property name="text">
          <string>Tipo:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="tipo_ref_cmb">
         <item>
          <property name="text">
           <string>Coluna</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Expressão</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2" colspan="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>249</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="tipo_ref_lbl_2">
         <property name="text">
          <string>Usado em:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="select_from_chk">
           <property name="text">
            <string>SELECT-FROM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="from_where_chk">
           <property name="text">
            <string>FROM-WHERE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="apos_where_chk">
           <property name="text">
            <string>Após WHERE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="tabela_lbl">
         <property name="text">
          <string>Tabela:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="alias_tab_lbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Alias Tabela:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="alias_tab_edt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="coluna_lbl">
         <property name="text">
          <string>Coluna:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="alias_col_lbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Alias Coluna:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="alias_col_edt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="expressao_lbl">
         <property name="text">
          <string>Expressão:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QTextEdit" name="expressao_txt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="alias_exp_lbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Alias Expressão:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="alias_exp_edt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="5" column="3" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>203</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Previsão de Código</string>
      </attribute>
      <layout class="QGridLayout" name="previsao_grid">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextEdit" name="codigo_txt">
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
          </font>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
